ddtrace/_trace/trace_handlers.py

```diff
@@ -688,10 +681,6 @@ def _on_botocore_patched_bedrock_api_call_success(ctx, reqid, latency, input_tok
     span = ctx.span
     span.set_tag_str("bedrock.response.id", reqid)
     span.set_tag_str("bedrock.response.duration", latency)
```

Comment:
```suggestion
```
I think we can actually just remove this function `_on_botocore_patched_bedrock_api_call_success` period, let's get rid of all these tags